src/CuArrays.jl

```diff
@@ -4,7 +4,7 @@ using CUDAapi, CUDAdrv, CUDAnative
 
 using GPUArrays
 
-export CuArray, CuVector, CuMatrix, CuVecOrMat, cu
+export CuArray, CuVector, CuMatrix, CuVecOrMat, cu, cuda
```

Comment:
seems like a pretty generic function to export (both `cu` and `cuda` are bound to confuse users). why not something that implies its action, e.g., `on_cuda`? or `@cuda` re `@async`?

Reply:
I agree about not exporting this for now. In the longer term, if this is successful it should replace `cu` entirely (alongside all the other APIs, for most users), so a generic name seems appropriate.

I think `cuda() do ...` reads right, and provides an obvious space for options (`cuda(device=2) do ...`), but `@cuda` could work well too (especially in that it's a bit nicer for one liners).